build.sh

```diff
@@ -58,7 +58,7 @@ BUILD_TYPE=Release
 INSTALL_TARGET=install
 BUILD_ALL_GPU_ARCH=0
 SINGLEGPU_CPP_FLAG=""
-SINGLEGPU_PYTHON_FLAG=""
+BUILD_PYTHON_ARGS=${BUILD_PYTHON_ARGS:=""}
```

Comment:
Maybe this should be a command-line flag instead, like "--python-args"? Seems like that would be more consistent with the other inputs

Reply:
I can try and convert it to a command line argument, but it will be a little riskier since it will be a little different than all of our current command line args. This would be the only one that accepts an argument value. All of the current options (`-g`, `-n`, `--singlegpu`, etc.) are boolean flags which is much easier to parse. I'm just a little concerned that we may miss some edge cases when parsing arbitrary arguments since this runs on several platforms.

I'll give it a shot and see if there are any other `build.sh` scripts in rapids that take arbitrary values

Reply:
I ended up sticking with boolean flags and added a new `--codecov` option to `build.sh`. This doesn't add any risk from parsing argument values and is generic enough to possibly include C++ code coverage one day (if we can figure out how to do it).

I'm open to different argument name. Let me know if you have a better selection. Link to the help section: https://github.com/rapidsai/cuml/pull/3111/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R46-R47